docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,395 @@
+    url: "https://github.com/nautls/nautilus-wallet",
+  },
+  {
+    id: "evernode",
+    label: "Evenode",
+    url: "https://evernode.org",
+  },
+  {
+    id: "raised-in-space",
+    label: "Raised in space",
+    url: "https://raisedinspace.com",
+  },
+];
+
+const articles = [
+  {
+    time: "JAN 2024",
+    title: "XRP Price Prediction: Will Ripple Make Waves in 2024?",
+    url: "https://www.forbes.com/advisor/au/investing/cryptocurrency/xrp-price-prediction",
+  },
+  {
+    time: "JAN 2024",
+    title: "XRP Could Surge Massively in 2024",
+    url: "https://u.today/xrp-could-surge-massively-at-beginning-of-2024-heres-why",
+  },
+  {
+    time: "NOV 2023",
+    title: "NFTs and the XRPL: A Marriage of Art and Technology",
+    url: "https://medium.com/@MariaSolorzano/title-nfts-and-the-xrpl-a-marriage-of-art-and-technology-cf76a0432693",
+  },
+];
+
+const FeaturedProjects = () => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const handlePrev = () => {
+    if (currentIndex > 0) {
+      setCurrentIndex(currentIndex - 1);
+    }
+  };
+
+  const handleNext = () => {
+    if (currentIndex < projects.length - 3) {
+      setCurrentIndex(currentIndex + 1);
+    }
+  };
+
+  const ProjectCard = ({ project, index }) => (
+    <a
+      className={`col float-up-on-hover ${index % 2 === 0 ? "even" : "odd"}`}
+      target="_blank"
+      href={project.url}
+    >
+      <div className="project-logo d-flex justify-content-center align-items-center mb-8">
+        <img className={project.id} alt={project.label} />
+      </div>
+      <div className="project-name h5">{project.label}</div>
+    </a>
+  );
+
+  return (
+    <div className="featured-projects">
+      <div className="project-cards-container card-grid card-grid-3xN">
+        <ProjectCard project={projects[currentIndex]} index={currentIndex} />
+        <ProjectCard
+          project={projects[currentIndex + 1]}
+          index={currentIndex + 1}
+        />
+        <ProjectCard
+          project={projects[currentIndex + 2]}
+          index={currentIndex + 2}
+        />
+      </div>
+      <div className="arrow-wrapper d-flex justify-content-center mt-16">
+        <button
+          className={`arrow-button prev ${
+            currentIndex > 0 ? "hover-color" : ""
+          }`}
+          onClick={handlePrev}
+        >
+          <img alt="left arrow" />
+        </button>
+        <button
+          className={`arrow-button next ${
+            currentIndex < projects.length - 3 ? "hover-color" : ""
+          }`}
+          onClick={handleNext}
+        >
+          <img alt="right arrow" />
+        </button>
+      </div>
+    </div>
+  );
+};
+
+export default function Tokenization() {
+  const modalRef = useRef(null);
+
+  return (
+    <div className="landing page-tokenization">
+      <div className="position-relative"></div>
+      <div className="position-relative d-none-sm"></div>
+      <section className="container-new mt-16">
```

Comment:
The top margin here pushes the main context box down excessively in layout sizes medium and below. Suggested fix:

```suggestion
      <section className="container-new mt-lg-16">
```